cmake/IgnUtils.cmake

```diff
@@ -150,6 +155,79 @@ macro(ign_find_package PACKAGE_NAME)
   # Parse the arguments
   _ign_cmake_parse_arguments(ign_find_package "${options}" "${oneValueArgs}" "${multiValueArgs}" ${ARGN})
 
+  #------------------------------------
+  # Handle ticktock
+  # TODO(CH3): Remove on tock
+  if(${PACKAGE_NAME} MATCHES "^ign(ition)?-")
+    message(DEPRECATION
+      "\nThe use of 'ign-' and 'ignition-' prefixed libraries has been "
+      "deprecated in favor of 'gz-' and will be removed in the next version!\n"
+    )
+    set(PACKAGE_NAME_BAK ${PACKAGE_NAME})
```

Comment:
the `PACKAGE_NAME_BAK` variable is confusing. can we use `GZ_PACKAGE_NAME` in this code path for the `gz-*` name and leave `PACKAGE_NAME` unchanged until we've determined which package name is to be used?

Reply:
841fc7cd5c4150a8dd661d15e092c441e0efc2f7